languages/python/BitwardenClient/bitwarden_client.py

```diff
@@ -12,33 +12,22 @@ def __init__(self, settings: ClientSettings = None):
             settings_json = json.dumps(settings.to_dict())
             self.inner = bitwarden_py.BitwardenClient(settings_json)
 
-    def password_login(self, email: str, password: str) -> ResponseForPasswordLoginResponse:
-        result = self._run_command(
-            Command(password_login=PasswordLoginRequest(email, password))
-        )
-        return ResponseForPasswordLoginResponse.from_dict(result)
-
-    def get_user_api_key(self, secret: str, is_otp: bool = False) -> ResponseForUserAPIKeyResponse:
+    def access_token_login(self, access_token: str) -> AccessTokenLoginResponse:
```

Comment:
We're not returning anything on `access_token_login` on other language integrations, beyond an error type in the languages that require it.

This is because the return values are not really applicable to secrets manager:
```rust
pub struct AccessTokenLoginResponse {
    pub authenticated: bool,
    /// TODO: What does this do?
    pub reset_master_password: bool,
    /// Whether or not the user is required to update their master password
    pub force_password_reset: bool,
    two_factor: Option<TwoFactorProviders>,
}
```